docs/api/specs/desktopAgentCommunicationProtocol.md

```diff
@@ -0,0 +1,457 @@
+---
+id: desktopAgentCommunicationProtocol
+sidebar_label: Desktop Agent Communication Protocol 
+title: Desktop Agent Communication Protocol  (next)
+---
+
+# Desktop Agent Communication Protocol (DACP)
+
+:::info _[@experimental](../fdc3-compliance#experimental-features)_
+
+The Desktop Agent Communication Protocol (DACP) is an experimental feature added to FDC3 in 2.2. Limited aspects of its design may change in future versions and it is exempted from the FDC3 Standard's normal versioning and deprecation polices in order to facilitate any necessary change.
+
+:::
+
+The Desktop Agent Communication Protocol (DACP) constitutes a set of standardized JSON messages or 'wire protocol' that can be used to implement an interface to a Desktop Agent, encompassing all API calls events defined in the [Desktop Agent API](../ref/DesktopAgent.md). For example, the DACP is used by the [`@finos/fdc3` npm module](https://www.npmjs.com/package/@finos/fdc3) to communicate with Browser-Resident Desktop Agents or a connection setup via the [FDC3 Web Connection Protocol](./webConnectionProtocol).
+
+## Protocol conventions
+
+DACP messages are defined in [JSON Schema](https://json-schema.org/) in the [FDC3 github repository](https://github.com/finos/FDC3/tree/fdc3-for-web/schemas/api).
+
+:::tip
+
+TypeScript types representing all DACP and WCP messages are generated from the JSON Schema source and can be imported from the [`@finos/fdc3` npm module](https://www.npmjs.com/package/@finos/fdc3):
+
+```ts
+import { BrowserTypes } from '@finos/fdc3';
```

Comment:
Import statement should use double quotes

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85